### ISSUE TITLE
can't delete a page

### ISSUE THREAD

**Issue by vkoleman** [parsimony/parsimony#48]
I tried to create a block in page with a title ending with '_url'
Now I can't delete a page with a title, and a URL: (?<title>(.*)),
as the complete URL displayed is URL : http://localhost/(?

My index page's content is not displayed anymore within the backend.

**Comment by parsimony** [comment]
Hello,

I tried to do the same thing. 
In Parsimony, dynamic webpages are made with a dynamic parameter between < dynamic parameter >
i.e. with this type of writing (?< name >(.*))

Both regex parameter and SQL property are alpha-numeric dynamic parameters.
As well as being a simple dynamic property, a SQL property is related to the values ​​of a given SQL field.

![urlrewriting](https://f.cloud.github.com/assets/1277326/446494/f0c3dd78-b1d7-11e2-8393-de6d528aa1b1.png)

Is your regex really written like this "  (?(._))  " ?
Perhaps, the page you created was : " (?< url >(._)) " ?

Anyway, the fact that there is no <dynamic parameter> in your example can explain why you can't delete this page and your page's content is not displayed.

Can you give me more informations about the way you built this URL ?

**Comment by parsimony** [comment]
By posting this comment, I realize that the <> are not printed when there is no space < STH >

**Comment by parsimony** [comment]
So I re-edit your question:
"I tried to create a block in page with a title ending with '_url'
Now I can't delete a page with a title, and a URL: (?< title >(.*)),
as the complete URL displayed is URL : http://localhost/(?"

Is this page (?< title >(._)) a new page like the following example?
![urlrewriting2](https://f.cloud.github.com/assets/1277326/446749/0f76f65e-b1df-11e2-8876-14047882be5a.png)
or is this page (?< title >(._)) a "native" or by default page?

**Comment by vkoleman** [comment]
I was not playing with SQL yet, just trying to have a post on a page. I am still in a discovery phase of Parsimony.
Please find below my problematic page.
![bugparsimony](https://f.cloud.github.com/assets/4254000/453913/3b5b7bae-b32a-11e2-956f-46b4aac8fa11.png)

**Comment by parsimony** [comment]
I have just reproduced this bug. Your point is very interesting. 

![reproduction](https://f.cloud.github.com/assets/1277326/454115/f43f5112-b330-11e2-9113-4a7a5ac078ea.png)

In fact, in order to display pages by their titles, you must generate your regex via dynamic page options like the following example:
![urlrewriting2](https://f.cloud.github.com/assets/1277326/454127/67ca4e98-b331-11e2-844a-0a8226cfe7d6.png)

After doing that, you must drag n drop a Query block in your page.

I have just done a video for you :)
Here is the link: http://www.youtube.com/watch?v=RePPlqdp4Gw

PS 
In my example, I chose the "title" parameter like your problematic.
But it's better to play with the "url" parameter that rewrites automatically your title with - instead of white space 
for instance my first post = my-first-post
I recommend you reinstall Parsimony because this bug is not fixed.
